### DISCUSSION TITLE
Website custom domain

### DISCUSSION THREAD

**Discussion by codeMaster-2004** [community/community#144476] [Actions]
### Select Topic Area

General

### Body

I have a website deployed on github with default domain name that I changed into custom domain. The issue is, When I change it to the custom domain , the website doesnt show up but when I open the default linked website it works. And the build and deployment logs of the website after I added the custom domain also show a successful build and deployment. 

But after I add the custom domain, the dns check runs for a while and approves with a green checkmark. But as soon as I refresh the page the dns checker runs again on the custom domain. is this normal?? if not what can I do to improve the situation. 
my godaddy dns configurations regarding the github website:

a	@	185.199.108.153	1 Hour		

a	@	185.199.109.153	1 Hour		

a	@	185.199.110.153	1 Hour		

a	@	185.199.111.153	1 Hour

cname	www	codemaster-2004.github.io.	1 Hour

there's also a CNAME file at the root of the gh-pages and main branch that has the custom domain's name:
analyticalscientificproducts.com

**Comment by OGcasscond** [comment]
Hi, I have a maybe dumb question - I have all of the code imported from a website I was previously paying for and successfully imported it to my github domain, but how the heck do I see it published?? I can only see the code when I look up my domain name
